autogpt/memory/__init__.py

```diff
@@ -34,6 +34,14 @@
     MilvusMemory = None
 
 
+try:
+    from autogpt.memory.llama_index import LlamaIndexMemory
+    supported_memory.append("llama_index")
+except ImportError:
+    print("llama-index not installed. Skipping import.")
```

Comment:
please comment this print line, it has been removed from the other imports as people complained it was an error

Reply:
done!